bamboo/Dockerfile

```diff
@@ -8,5 +8,5 @@ xz-utils tk-dev libffi-dev liblzma-dev python3-pip
 RUN update-alternatives --install /usr/bin/python python /usr/bin/python3.7 1 && \
     update-alternatives --install /usr/bin/pip pip /usr/bin/pip3 1
 RUN python -m pip install -U pip
-RUN npm install -g npm@6.14 && npm config --global set script-shell /bin/bash
+RUN npm install -g npm@8.6.0 && npm config --global set script-shell /bin/bash
```

Comment:
Do we want to keep the patch version off like before? What was the thought of adding it in?

Reply:
Hmm, that's fair.    I was erring on the side of being explicit, but there's no reason we can't take patches by default. .

Reply:
419d186